### PR TITLE
Add exit_unknown_on_exception decorator.

### PR DESCRIPTION
**Pull request by andrekeller**
The decorator will cause the program to exit with error code 3
and output the string representation of the exception on stdout.

### CODE REVIEW

File: simple_icinga_plugin/__init__.py [hunk 1]
```diff
@@ -7,7 +7,6 @@
 __author__ = "Andre Keller"
 __copyright__ = "Copyright (c) 2015, VSHN AG, info@vshn.ch"
 __license__ = 'BSD'
-__version__ = '0.1.0'
```
Comment (hansmi) [inline simple_icinga_plugin/__init__.py]: Unrelated change.

**Review by hansmi — COMMENTED** [review]
One issue, rest LGTM.

File: simple_icinga_plugin/__init__.py (hunk 1 above)
Comment (andrekeller) [reply simple_icinga_plugin/__init__.py]: yeah, that one slipped through...
